dcpy/utils/geospatial.py

```diff
@@ -166,3 +166,18 @@ def read_parquet_metadata(filepath: Path) -> geoparquet.MetaData:
     geo = parquet_metadata.metadata[geoparquet.GEOPARQUET_METADATA_KEY]
     geo_parquet = geoparquet.GeoParquet(**json.loads(geo))
     return geoparquet.MetaData(file_metadata=parquet_metadata, geo_parquet=geo_parquet)
+
+
+def reproject_gdf(
+    df: gpd.GeoDataFrame, target_crs: str, source_crs: str | None = None
+) -> gpd.GeoDataFrame:
+    if not df.crs:
```

Comment:
Not sure if this is needed here. You can't read data into a gpd data frame if it's not geospatial

Reply:
You can certainly have geometry data without having a crs set. Not geoparquet, so we'll be safe when reading from that, but in general [gdf.crs can return None](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoDataFrame.crs.html)

Reply:
oh so it's probably similar to crs not being set on a geometry column in Postgres in which case it would return zero. Thank you for clarifying